docs/planning/sprintplanning.md

```diff
@@ -0,0 +1,66 @@
+# Knative Eventing sprint planning
+
+This document summarizes project planning mechanics for Knative Eventing releases. Knative Eventing release is planned every 6 weeks and will continue to be so. Current proposal is to align sprint with the release cycle of 6 weeks. Eventually this may change as we learn from the process.
+
+## How to manage Bugs?
+- Submit new bugs by creating a new issue of type “Report Bug”.
+- Please fill in as much detail as possible in the bug template and propose priority and severity
+- All new bugs will be labelled “triage/needstriage” and kind/bug as part of the template
+- Bugs will be triaged weekly (this could be modified based on incoming bugs rate). And upon triage will be labeled triage/accepted, or triage/rejected (and closed) with appropriate priority (priority/*) and severity (severity/*)
+- priority/p1 and priority/p2 bugs will be planned for next release cycle to keep low engineering debt.
+- priority/p0 is reserved for critical bugs that needs to be addressed immediately in current release at the cost of currently planned items.
+- WG lead will drive this and his/her decision will be final
+
+## How to manage Story/Feature?
+- Story/Feature typically spans 1-2 weeks for one engineer.
+To propose a new feature,  create a new issue of type “Feature Request”
+- Please fill in as much detail as possible in the Feature template and propose a release cycle
+- All new features will be labelled “kind/feature”, and “triage/needstriage”
```

Comment:
Do we need two labels here, or just `kind/feature`?

Reply:
(and then the milestone label can be used when they are accepted)

Reply:
Will summarize and send a new PR with minimal set of labels based on our discussion